server/src/main/java/eu/fasten/server/FastenServer.java

```diff
@@ -54,7 +51,17 @@
             defaultValue = ".")
     private Path pluginPath;
```

Comment:
Can we also add a `-pl`, `--list_plugins` option that reads displays a table with meta information about the loaded plug-ins? It can look like this:

```
PLUGIN, VERSION, DEFAULT_TOPIC, DESCR
OPAL, 0.0.1, fasten.mvn.packages, Reads a Maven coord and creates a callgraph
...
```

Reply:
This is very possible. I also though that there can be an argument to specify names of plugins that we want to run. So we don't have to maintain a list of different docker images with different plugins in each, we will just end up having one Docker image with all plugins, but when at the start of the container we'll specify plugins that need to run and the metadata for those will be automatically shown. So `-pl OPAL, MetadataDBExtension` will only run those two plugins and also automatically show metadata for them.

Reply:
Yes, makes sense. We will have a single Docker image in any case after @wzorgdrager's changes. Let's do that.

Reply:
Now we can specify a number of plugins to run using `-pl` (can be repeated). Adding `-pl OPLA` will end up stopping and unloading all other plugins and printing OPAL's metadata to log. The only difference is that default_topic is not included in this metadata, as it's a generic way of printing info for FastenPlugins, but topic is KafkaPlugins-specific